src/libclient/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+project(libowncloudclient)
```

Comment:
how about  `libsyncclient`?
(maybe we can rename the other lib in `libsyncengine`